packages/coin-algo/src/utils/scriptUtils.ts

```diff
@@ -0,0 +1,77 @@
+import { coin as COIN, error as ERROR, utils, config } from '@coolwallet/core';
+import nacl from "tweetnacl";
+import base32 from "hi-base32";
+import sha512 from "js-sha512";
+import * as params from '../config/params';
+import * as types from '../config/types';
+import { getTransactionArgument } from "./transactionUtils";
+const msgpack = require('algo-msgpack-with-bigint');
+
+const PUBLIC_KEY_LENGTH = nacl.sign.publicKeyLength;
```

Comment:
Ed25519 public key length will always be 32 bytes, I think we might be able to drop `tweetnacl` in dependencies.